Formula/llvm.rb

```diff
@@ -95,66 +125,95 @@ class Llvm < Formula
       url "http://llvm.org/git/lldb.git"
     end
 
-    # Polly is --HEAD-only because it requires isl and the version of Polly
-    # shipped with 3.6.2 only compiles with isl 0.14 and earlier (current
-    # version is 0.15). isl is distributed with the Polly source code from LLVM
-    # 3.7 and up, so --HEAD builds do not need to depend on homebrew isl.
-    option "with-polly", "Build with the experimental Polly optimizer"
     resource "polly" do
       url "http://llvm.org/git/polly.git"
     end
   end
 
-  keg_only :provided_by_osx
+  # keg_only :provided_by_osx if OS.mac?
 
-  option :universal
-  option "with-clang", "Build the Clang compiler and support libraries"
+  # Options
+  option :universal if OS.mac?
+  option "with-clang",             "Build the Clang compiler and support libraries"
   option "with-clang-extra-tools", "Build extra tools for Clang"
-  option "with-compiler-rt", "Build Clang runtime support libraries for code sanitizers, builtins, and profiling"
-  option "with-libcxx", "Build the libc++ standard library"
-  option "with-lld", "Build LLD linker"
-  option "with-lldb", "Build LLDB debugger"
-  option "with-python", "Build Python bindings against Homebrew Python"
-  option "with-rtti", "Build with C++ RTTI"
-  option "with-utils", "Install utility binaries"
+  option "with-compiler-rt",       "Build Clang runtime support libraries for code sanitizers, builtins, and profiling"
+  option "with-libcxx",            "Build the libc++ standard library"
+  option "with-libcxxabi",         "Build the libc++abi standard library"
+  option "with-libunwind",         "Build the libunwind library"
+  option "with-lld",               "Build LLD linker"
+  option "with-lldb",              "Build LLDB debugger"
+  option "with-rtti",              "Build with C++ RTTI"
+  option "with-utils",             "Install utility binaries"
+  option "with-polly",             "Build with the experimental Polly optimizer"
+  option "with-python",            "Build Python bindings against Homebrew Python"
+  option "with-test",              "Build LLVM unit tests"
+  option "with-shared-libs",       "Build all libs as shared instead of static"
+  option "with-libffi",            "Use libffi to call external functions from the interpreter"
 
   deprecated_option "rtti" => "with-rtti"
 
-  if MacOS.version <= :snow_leopard
+  # Dependencies
+  depends_on "cmake"    => :build
```

Comment:
Please don't introduce spacing gaps in the `:build` style tags. A lot of editors just complain about it and it's not that much more aesthetically pleasing.